x-pack/plugins/apm/public/components/shared/transactions_table/index.tsx

```diff
@@ -88,7 +63,7 @@ export function TransactionsTable({
   hideViewTransactionsLink = false,
   hideTitle = false,
   isSingleColumn = true,
-  numberOfTransactionsPerPage = 5,
+  numberOfTransactionsPerPage = 10,
```

Comment:
Why did you change the default value here?

Reply:
I made the default 10 because that's what we show on the error overview. Then on the service overview we explicitly pass in `numberOfTransactionsPerPage={5}`.